packages/shadcn/src/lib/utils.ts

```diff
@@ -1,5 +1,9 @@
 import { type ClassValue, clsx } from "clsx";
-import { twMerge } from "tailwind-merge";
+import { extendTailwindMerge } from "tailwind-merge";
+
+const twMerge = extendTailwindMerge({
```

Comment:
Why do we need this? Aren't all classnames we pass to cn already prefixed?

Reply:
Yes, this is just so that `twMerge` recognizes that the `bn-` prefixed Tailwind classes are in fact still Tailwind classes, otherwise it treats them like regular classes so it doesn't combine and order them properly.

e.g. if you do `twMerge("w-10", "w-16")` you'll get `"w-16"`, but `twMerge("bn-w-10", "bn-w-16")` you'll get `"bn-w-10 bn-w-16"` unless you extend it to recognize the `bn-` prefix.

Reply:
clear! maybe add this as a comment